data_structures/binary_tree/binary_tree_traversals.py

```diff
@@ -86,10 +86,10 @@ def printLeftToRight(root,level):
     if level==1:
         print(root.data,end=" ")
     elif level>1:
-        printLeftToRight(root.left,level-1)
-        printLeftToRight(root.right,level-1)
+        printlefttoright(root.left,level-1)
+        printlefttoright(root.right,level-1)
 
-def printRightToLeft(root,level):
+def printrighttoleft(root,level):

```

Comment:
```suggestion
def print_right_to_left(root,level):
```
Don't be so hostile to your reader.

Reply:
This one can print() because it is not an algorithmic function.

Reply:
sorry i did'nt get what you are suggesting ?